### ISSUE TITLE
Retire AppVeyor, ensure GitHub actions has feature parity

### ISSUE THREAD

**Issue by seanbudd** [nvaccess/nvda#17878]
### Related issues, PRs or discussions
Follow up to #17728 and #10516

### What is the current state of the codebase?

After #17728 is merged, NVDA is now being built with GitHub actions in parallel to AppVeyor.
It is missing key parts of AppVeyor, many required to create a release build of NVDA:

- Features:
  - feature_signing: code signing
  - feature_buildSymbols: creating build syms for NVDA
  - feature_uploadSymbolsToMozilla: uploading symbols to mozilla
  - feature_crowdinSync: uploading translations to Crowdin
- Deploying the release via the NV Access server hook
- Uploading various artifacts

### Why are changes required?

We want parity between GitHub actions and AppVeyor before migrating over permanently

### What technical changes are required?

- Going over the AppVeyor build scripts and ensuring feature parity in GitHub actions (see disabled features)
- Comparing the artifacts of PR builds, alpha/beta/rc branch builds and tagged release builds
- Testing a tagged release of NVDA
- Moving AppVeyor documentation over to the GitHub actions docs
- Removing AppVeyor scripts
- Turning off AppVeyor integration through the web UX, removing it from access to our repository

### Are the proposed technical changes API breaking?
No
### Are there potential risks or issues with the proposed implementation?
No

**Comment by seanbudd** [comment]
Closing as complete
